python/az/aro/azext_aro/_help.py

```diff
@@ -12,31 +12,53 @@
 helps['aro create'] = """
     type: command
     short-summary: Create a cluster.
+    examples:
+      - name: Create an OpenShift cluster
```

Comment:
```suggestion
      - name: Create an OpenShift cluster.
```

All of these should end with `.`.